app/messages/Messages.scala

```diff
@@ -404,6 +404,71 @@ object Messages {
 
   }
 
+  object Vat {
+    val `Set up a VAT payment plan`: Message = Message(
+      english = "Set up a VAT payment plan"
+    )
+    val `The payment plan covers all appropriate overdue amounts...`: Message = Message(
+      english = "The payment plan covers all appropriate overdue amounts, surcharges, penalties and interest. The payments you make may incur interest."
+    )
+    val `Who can use this service`: Message = Message(
+      english = "Who can use this service"
+    )
+    val `You are eligible to set up an online payment plan if:`: Message = Message(
+      english = "You are eligible to set up an online payment plan if:",
+      welsh   = "Rydych yn gymwys i drefnu cynllun talu ar-lein os:"
+    )
+    def `you plan to pay the debt off within the next 6 months or less`(maxPlanDuration: Int): Message = Message(
```

Comment:
small thing but `6` in the val name doesn't reflect the input to this `def` - I've put `...` in similar `val` names for EPAYE where the value is variable

Reply:
ah ok! Will update